packages/main/src/StepInput.ts

```diff
@@ -326,6 +326,15 @@ class StepInput extends UI5Element implements IFormElement {
 
 	_initialValueState?: `${ValueState}`;
 
+	/**
+	 * Indicates if the component is initially rendering.
+	 * It's used to round the value of 0 once to illustrate `valuePrecision` property effect.
+	 * Subsequent value changes aren't auto-rounded to preserve user input accuracy.
+	 *
+	 * @private
+	 */
+	_initialRender!: boolean;
```

Comment:
I wonder, can we always round the 0, because if the preson deletes everything it will go to zero, so it will be kind of initial render, even tho it's not.  We can evaluate the cases when we will have 0 as a value

Reply:
Hm, Ive thought about that too, and tested it, but it would precise the number always when it is 0, which doesnt match fully the idea of removing the rounding and wont be consistent with the other values ? Maybe lets leave it like that ?

Reply:
Hi colleagues, we discussed it offline, and we decided that we would precise the value when it is 0 when:

- User deletes the whole input manually;
- When the initial value of the component is 0 (explicitly set) or not set at all (which is 0 by default);